atlp-rwanda/e-commerce-lydia-32-fn
# Users should be able to Login with Email and Password

### What does this PR do?
This PR implements the login functionality using email and password on the frontend.
### Description of Task to be completed?
Develop the frontend logic for email and password authentication.
### How should this be manually tested?
Enter a valid email and password, then click "Login".

- Verify that a successful login redirects to the home page and a success toast appears.
